Dockerfile

```diff
@@ -44,9 +44,7 @@ FROM base AS criu
 # Install CRIU for checkpoint/restore support
 ENV CRIU_VERSION 3.6
 # Install dependancy packages specific to criu
-RUN case $(uname -m) in \
```

Comment:
@tianon pinged me that we should avoid `uname -m`; this removes one, but https://github.com/moby/moby/pull/36546 added another one below

@tianon what do you suggest?

Reply:
I'd suggest something based on the arch of the image, not the kernel that happens to be running it.  If you've got `go`, then `go env GOARCH` can be sufficient (depending on the ARM granularity required), but I usually use `dpkg --print-architecture` or `apk --print-arch` (depending on the underlying platform and whether I want to `apk add --no-cache dpkg` on Alpine).

Reply:
hmm~~, so sweat suggestion 🥇 ! I will replace another one with `dpkg --print-architecture` after this PR. (a `syscall.dup2` issue for `manifest schema1` on arm prevent us to remove all)